pkg/sink/codec/canal/canal_json_txn_encoder.go

```diff
@@ -62,6 +62,7 @@ func (j *JSONTxnEventEncoder) AppendTxnEvent(event *commonEvent.DMLEvent) error
 		}
 		value, err := newJSONMessageForDML(&commonEvent.RowEvent{
 			TableInfo:      event.TableInfo,
+			StartTs:        event.StartTs,
```

Comment:
start-ts is not encoded into the canal-json message, so no need to set it, make it confuse.